stock_move_actual_date/models/actual_date_mixin.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2024 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ActualDateMixin(models.AbstractModel):
+    _name = "actual.date.mixin"
```

Comment:
A little bit `stock` context in name or is this intended to be used in other flows ?

Reply:
@rousseldenis Our purpose is limited to the stock scope for this model, so I changed it to `stock.actual.date.mixin`.
Thanks for the suggestion.